preview/Dockerfile

```diff
@@ -1,5 +1,16 @@
 FROM buildpack-deps:trusty-scm
 
+# Temporarily work around https://github.com/dotnet/cli/issues/1582 by
+# avoiding the restart_syscall made by LTTng which causes a failed assertion.
+# This workaround specifies that applications should not wait for a sessiond
+# "registration done" command before proceeding to execute the main program.
+#
+# This issue is being addressed in Docker (see
+# https://github.com/docker/docker/issues/20818). When that fix has been
+# included in a release of Docker, this workaround will be removed, and users
+# will be expected to upgrade their Docker installation.
```

Comment:
In a way I think this is a little verbose but that is just my opinion.  One thing that is incorrect is to say users are "expected to upgrade".  Users can use the seccomp functionality to either run in unconfined mode or with a custom profile that allows the restart_syscall.  The reason we are doing this workaround is to simply provide the best out of box experience for those who want to kick the tires.

I think you could get away with simply saying "Workaround https://github.com/dotnet/cli/issues/1582 until Docker releases a fix (https://github.com/docker/docker/issues/20818).  This workaround allows the container to be run with the default seccomp docker settings."